dmcontent/content_loader.py

```diff
@@ -199,7 +205,7 @@ def get_question_as_section(self, question_slug):
             editable=self.edit_questions,
             edit_questions=False,
             questions=question.get('questions', [question]),
-            description=question.get('hint'),
+            description=question.get('hint') if question.get('questions') else '',
```

Comment:
Just trying to work out why we need this?  So only multiquestions display their hint as the description, but singular-questions don't?